Firmware/skr_mini_e3_v3_config.cfg

```diff
@@ -0,0 +1,403 @@
+endstop_pin: probe:z_virtual_endstop
+position_max: 250
+homing_speed: 40
+position_min: -3.0
+
+[tmc2209 stepper_z]
+uart_pin: PC11
+tx_pin: PC10
+uart_address: 1
+run_current: 0.5
+interpolate: False
+stealthchop_threshold: 0
+
+#####################################################################
+#   Extruder Settings
+#####################################################################
+
+######
+#Motor - EM
+###############
+[extruder]
+# E0_STEP_PIN        PB3
+# E0_DIR_PIN         PB4
+# E0_ENABLE_PIN      PB1
+# E0_UART_RX         PC11
+# E0_UART_TX         PC10
+step_pin: PB3
+dir_pin: PB4
+enable_pin: !PD1
+# Tune per individual printer
+# Default for Bondtech 5mm Bore Drive Gears
+rotation_distance: 22.6789511
+# Tune for extruder 
+gear_ratio: 50:17
+microsteps: 32
+full_steps_per_rotation: 200
+nozzle_diameter: 0.400
+filament_diameter: 1.75
+heater_pin: PC8
+##  Validate the following thermistor type to make sure it is correct
+##  See https://www.klipper3d.org/Config_Reference.html#common-thermistors for additional options
+# sensor_type: Generic 3950
+sensor_pin: PA0
+min_temp: 10
+max_temp: 270
+max_power: 1.0
+min_extrude_temp: 170
+control = pid
+pid_kp = 26.213
+pid_ki = 1.304
+pid_kd = 131.721
+#Set appropriate once tuning your printer
+#pressure_advance: .05
+##	Default is 0.040, leave stock
+# pressure_advance_smooth_time: 0.040
+max_extrude_only_distance: 100.0
+
+[tmc2209 extruder]
+uart_pin: PC11
+tx_pin: PC10
+uart_address: 3
+run_current: 0.7
+hold_current: 0.3
+interpolate: False
+
+#####################################################################
+# 	Bed Heater
+#####################################################################
+
+######
+# BED Connector
+###############
+[heater_bed]
+heater_pin: PC9
+##  Validate the following thermistor type to make sure it is correct
+##  See https://www.klipper3d.org/Config_Reference.html#common-thermistors for additional options
+# sensor_type: Generic 3950
+sensor_pin: PC4
+min_temp: 0
+max_temp: 130
+control: pid
+pid_kp: 58.437
+pid_ki: 2.347
+pid_kd: 363.769
+
+#####################################################################
+# 	Probe
+#####################################################################
+
+######
+#Z Max Connector on Z(main) Board
+#Inductive Probe
+###############
+[probe]
+##      If your probe is NO instead of NC, add change pin to !z:P1.24
+pin: ^PC2
+x_offset: 0
+y_offset: 25
+z_offset: 0
+samples: 3
+samples_result: median
+sample_retract_dist: 3
+samples_tolerance: 0.006
+samples_tolerance_retries: 3
+
+#####################################################################
+# 	Fan Control
+#####################################################################
+
+######
+# Electronics Fan
+# FAN1 Connector
+###############
+[controller_fan my_controller_fan]
+pin: PC7 
+max_power: 1.00
+kick_start_time: 0.200
+heater: heater_bed
+
+######
+# Hot End Fan
+# FAN2 Connector
+###############
+[heater_fan extruder_fan]
+pin: PC15
```

Comment:
This pin is incorrect

Reply:
Good catch, corrected [9a7c9eb](https://github.com/VoronDesign/Voron-Switchwire/pull/44/commits/9a7c9ebe65667b751fb2e27f25636313386a9d0c)